script_runner/tasks.py

```diff
@@ -348,6 +348,7 @@ def _get_process_environment(process, proxy):
     the current executable.
     """
     env = os.environ.copy()
+    env.setdefault('TMPDIR', get_exec_tempdir())
```

Comment:
`TMPDIR` is only used by Python.
A shell script calling something like `my_file=$(mktemp)` won't use `TMPDIR`, will it? I think it'd only use `TEMP` or `TMP`.

Reply:
@isaac-s `TMPDIR` is used by `mktemp` as per the manual. Possibly not all systems, but it is used on all linuxes I tried it on